policyengine_us/variables/gov/states/la/tax/income/deductions/la_itemized_deductions.py

```diff
@@ -0,0 +1,36 @@
+from policyengine_us.model_api import *
+
+
+class la_itemized_deductions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Louisiana itemized deduction"
+    unit = USD
+    definition_period = YEAR
+    reference = [
+        "https://revenue.louisiana.gov/TaxForms/IT540iWEB(2022)D1.pdf",
+        "https://www.legis.la.gov/Legis/Law.aspx?d=101760",  # (3)
+    ]
+    defined_for = StateCode.LA
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
```

Comment:
Can you leave some comments here? I just took a quick scan but I don't think this matches my understanding. Looking at the forms, it seems like the change from 2021 to 2022 is that the excess itemized deductions are now limited to the medical expense deduction, is that right? If that's how you're implementing it, the term `availability` implies an expansion rather than limitation of the deduction.

Reply:
The comments you left are still not clear.

Reply:
we are attempting to represent the condition that since 2022 Louisiana computes the itemized deduction as the excess of the federal medical expense deduction over the federal standard deduction: [Law](https://www.legis.la.gov/Legis/Law.aspx?d=101760)
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/7853fe81-e892-4457-bb16-e73b4094d86d)
In 2021 and prior the itemized deductions were computed by subtracting the federal standard deduction from the federal itemized deduction: [law 2021](https://law.justia.com/codes/louisiana/2020/revised-statutes/title-47/rs-293/)
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/acf76785-ae28-4ff3-8ef1-32808a6fd2db)

Let me know if we are misinterpreting the law here

Reply:
thanks - updated code suggestions accordingly